consoles/blackbox-overview.html

```diff
@@ -0,0 +1,54 @@
+{{ template "head" . }}
+
+{{ template "prom_right_table_head" }}
+{{ $job := .Params.job }}
+<tr>
+  <th>{{ $job }}</th>
+  <th>{{  template "prom_query_drilldown" (args (printf "sum(up{job='%s'})" $job))  }} / {{ template "prom_query_drilldown" (args (printf "count(up{job='%s'})" $job)) }}</th>
+</tr>
+
+<tr>
+  <th colspan=2>Currently</th>
+</tr>
+{{ range query (printf "probe_success{job='%s'}" $job) | sortByLabel "instance" }}
+<tr>
+  <td {{ if eq (. | value) 1.0 }}{{ else }} class="alert-danger"{{ end }}>
```

Comment:
OK, this is a style thing I was wondering about. I think changing the colour across the whole row is better in the current config.  But the current config is surprising - it only changes colour when there's something wrong. There's no indication colour is a thing in these tables till something breaks. I was thinking we might want to add "alert-success" when there's a yes and in that case just changing the cell would look nicer.

So... I'll remove this bit here for now so it matches everything else and I'll send in a separate CL for all the pages to either "alert-danger" the tr or to "alert-success" the cell when Yes/Up. Sound good?